telegram-bot/pom.xml

```diff
@@ -40,12 +40,13 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>31.1-jre</version>
+            <version>${guava.version}</version>
```

Comment:
what is the need for guava?

Reply:
No idea atm. Presumably, it was used in tests or as additional lib, but i don't remember why and when.
Googled and still no idea. Deleted, tests and project work.